src/lib/daemon_rpcs/types.ml

```diff
@@ -250,6 +250,8 @@ module Status = struct
     let block_production_keys =
       list_string_entry "Block producers running" ~to_string:Fn.id
 
+    let coinbase_receiver = string_option_entry "coinbase receiver"
```

Comment:
Should this be init-capped, as for the other status items?

Reply:
I'm not sure what this means

Reply:
"Coinbase receiver"